examples/counter/Cargo.toml

```diff
@@ -20,10 +20,10 @@ rust-version = "1.66"
 
 [workspace.dependencies]
 anyhow = "1.0.83"
-crux_core = "0.7"
-crux_http = "0.9"
-# crux_core = { path = "../../crux_core" }
-# crux_http = { path = "../../crux_http" }
```

Comment:
Committed inadvertently?

Reply:
kinda. Was using to test. What I really want is them to point to local paths when on a PR, and to the published versions when on master. 😓 Will revert